webapi/src/test/scala/org/knora/webapi/e2e/v2/SearchRouteV2R2RSpec.scala

```diff
@@ -7331,5 +7331,30 @@ class SearchRouteV2R2RSpec extends R2RSpec {
 
         }
 
+        "reject a link value property in a query in the simple schema" in {
+            val gravsearchQuery =
+                """
+                  |PREFIX incunabula: <http://0.0.0.0:3333/ontology/0803/incunabula/simple/v2#>
+                  |PREFIX knora-api: <http://api.knora.org/ontology/knora-api/simple/v2#>
+                  |
+                  |CONSTRUCT {
+                  |    ?book knora-api:isMainResource true .
+                  |    ?book incunabula:title ?title .
+                  |    ?page incunabula:partOfValue ?book .
+                  |} WHERE {
+                  |    ?book a incunabula:book .
+                  |    ?book incunabula:title ?title .
+                  |    ?page a incunabula:page .
+                  |    ?page incunabula:partOfValue ?book .
+                  |}
+                """.stripMargin
+
+            Post("/v2/searchextended", HttpEntity(SparqlQueryConstants.`application/sparql-query`, gravsearchQuery)) ~> addCredentials(BasicHttpCredentials(incunabulaUserEmail, password)) ~> searchPath ~> check {
+
+                assert(status == StatusCodes.NOT_FOUND, response.toString)
```

Comment:
What does the error message say now exactly? Does it say that the property `incunabula:partOfValue` was not found since it was added to `propertiesUnavailableInSchema`?

Would it make sense to check that the error message contains that information?

Reply:
It says:

```
Some requested entities were not found: http://0.0.0.0:3333/ontology/0803/incunabula/simple/v2#partOfValue
```

I added a check for the property IRI in the error message.